src/server/Storages/usersStore.ts

```diff
@@ -0,0 +1,24 @@
+import "sequelize";
+import { User } from '../../lib/model';
+import { sequelize } from "../database/database";
+import { MUser } from "../database/models/MUser";
+
+export function getAll(): Promise<User[]> {
+    return sequelize.sync()
```

Comment:
`sequelize.sync()` should be called once inside [server.ts](https://github.com/edu-xored/calendar/blob/master/server.ts) before starting HTTP server.